problems/arrayMethods.js

```diff
@@ -91,15 +141,33 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
+const counterObject = (items) => {
+    let obj = {}
+    items.forEach((el) => {
+      if (obj[el]) {
+        obj[el] += 1
+      } else {
+        obj[el] = 1
+      }
+    })
+    return obj 
+};
+console.log(counterObject([1, 1, 2, 1, 3]))
+console.log(counterObject(["cat", "bird", "dog", "cat"]))
 
 /**
  * Takes in an array of numbers and returns a new array with each element doubled.
  * @param {number[]} nums
  * @returns {number[]} Each element has been doubled
  */
 
-const doubled = () => {};
+const doubled = (arr) => {
+    let output = [];
+    arr.forEach((el) => {
```

Comment:
This should be a map because map returns an array which will eliminate the need to create a new array and push into it